home.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+  <title>Senior Swift - Concise weekly blog covering skills you need day to day that employers want</title>
+  <link rel="stylesheet" href="styles.css">
+
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playball|Playfair+Display:700" rel="stylesheet">
+</head>
+
+<body class="home-page">
+  <section class="home-banner">
+    <h1 class="blog-title">Swifty</h1>
+    <h2 class="blog-description">Weekly swift articles covering actual useful skills with short concise posts</h2>
+  </section>
+
+  <section class="blog-post-list">
+    <div class="container">
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/one.svg"/>
+        <a href="home.html" class="blog-post-title">First Blog Post</a>
+        <time datetime="2019-5-31">May 31 2019</time>
+      </div>
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/two.svg"/>
+        <a href="home.html" class="blog-post-title">Second Blog Post - A longer title</a>
+        <time datetime="2019-6-03">June 3 2019</time>
+      </div>
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/three.svg"/>
+        <a href="home.html" class="blog-post-title">Third Blog Post - The longest title we have had yet, I think</a>
+        <time datetime="2019-6-03">June 3 2019</time>
+      </div>
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/four.svg"/>
+        <a href="home.html" class="blog-post-title">Fourth Blog Post - No this is the longest because it is in fact a
+        stress test to see what happens when we have an overly large title. Do we cut it off, use elipsis or expand.</a>
+        <time datetime="2019-6-08">June 8 2019</time>
+      </div>
+    </div>
+  </section>
+  </div>
```

Comment:
Too many closing divs?
Also do you need all of these? If a wrapper won't contain anything that's not already inside the element it's wrapping?

Reply:
I've added container as a section class and removed the div.